evm_loader/program/src/evm/memory.rs

```diff
@@ -14,6 +14,7 @@ const MEMORY_ALIGN: usize = 1;
 
 static_assertions::const_assert!(MEMORY_ALIGN.is_power_of_two());
 
+#[repr(C)]
 pub struct Memory {
```

Comment:
In this file, `serde::Serialize` and `serde::Deserialize` impls can be removed.

Reply:
done